connectivity/check/deployment.go

```diff
@@ -413,6 +565,31 @@ func (ct *ConnectivityTest) validateDeployment(ctx context.Context) error {
 		}
 	}
 
+	perfPods, err := ct.client.ListPods(ctx, ct.params.TestNamespace, metav1.ListOptions{LabelSelector: "kind=" + kindPerfName})
+	if err != nil {
+		return fmt.Errorf("unable to list perf pods: %s", err)
+	}
+	for _, perfPod := range perfPods.Items {
+		ctx, cancel := context.WithTimeout(ctx, ct.params.ciliumEndpointTimeout())
+		defer cancel()
+		if err := ct.waitForCiliumEndpoint(ctx, ct.clients.src, ct.params.TestNamespace, perfPod.Name); err != nil {
+			return err
+		}
+		_, hasLabel := perfPod.GetLabels()["server"]
+		if hasLabel {
+			ct.perfServerPod[perfPod.Name] = Pod{
+				K8sClient: ct.client,
+				Pod:       perfPod.DeepCopy(),
+				port:      5201,
```

Comment:
From a high-level glance it's not clear to me why in this branch we put the port number, but in the else we omit it.

Reply:
Setting the port number is purely informational.